### PR TITLE
Withdraw liquidations

### PR DESCRIPTION
**Pull request by vladjdk**

### CODE REVIEW

File: src/components/free-willy/LiquidationWithdrawals.tsx [hunk 1]
```diff
@@ -1,343 +1,151 @@
...
+        pending_liquidated_collateral: number;
+        wait_end: number;
+        epoch_snapshot: number;
+        scale_snapshot: number;
     }
-
-    if (!connectedWallet.network.chainID.startsWith("columbus")) {
-      alert(`Please only execute this example on Mainnet`);
-      return;
+    interface BidsByUserResponse {
```
Comment (theodubiel) [inline src/components/free-willy/LiquidationWithdrawals.tsx]: I dont think this guy is used anymore

File: src/components/free-willy/LiquidationWithdrawals.tsx [hunk 2]
```diff
@@ -1,343 +1,151 @@
-import { MsgExecuteContract } from "@terra-money/terra.js";
-import { useConnectedWallet, useLCDClient } from "@terra-money/wallet-provider";
-import { useCallback, useEffect, useState } from "react";
-import BigNumber from "bignumber.js";
+import { useConnectedWallet, useLCDClient, useWallet } from "@terra-money/wallet-provider";
```
Comment (theodubiel) [inline src/components/free-willy/LiquidationWithdrawals.tsx]: clear out unused imports

File: src/components/free-willy/LiquidationWithdrawals.tsx [hunk 3]
```diff
@@ -1,343 +1,151 @@
...
-
-  const withdrawLuna = useCallback(() => {
-    if (!connectedWallet) {
-      return;
+    const network = useNetwork();
+    const {getFilledBidsPendingClaimAmount, claimLiquidations} = useAnchorLiquidationContract(network.contracts.anchorLiquidation);
+    
+    const connectedWallet = useConnectedWallet();
+    
+    interface Bid {
```
Comment (theodubiel) [inline src/components/free-willy/LiquidationWithdrawals.tsx]: You can import this from the hook file
`import { useAnchorLiquidationContract, Bid } from "hooks/useAnchorLiquidationContract";`

No need to redeclare interfaces

File: src/components/free-willy/LiquidationWithdrawals.tsx [hunk 4]
```diff
@@ -1,343 +1,151 @@
...
+                        })
+                        console.log(collaterals)
+                    })
+                } else {
+                    setCollaterals(null);
+                }
+            }, [connectedWallet, txInfo])
+            
+            return (
+                <div>
```
Comment (theodubiel) [inline src/components/free-willy/LiquidationWithdrawals.tsx]: You might be able to swap this `<div>` to a `<>`

**Review by 0xVillager — COMMENTED** [review]
Specifying 12 grid for xs (mobile) and half for medium and larger will make these responsive.

File: src/components/free-willy/FreeWilly.tsx [hunk 5]
```diff
@@ -2,25 +2,33 @@ import LiquidationWithdrawals from 'components/free-willy/LiquidationWithdrawals
...
-                </Paper>
-            </Stack>
+        <Container sx={{maxWidth: '1200px', padding: '10px'}}>
+            <Grid container spacing={2} >
+                <Grid item xs={12}>
+                    <Paper elevation={3}>
+                        <LiquidationBidChart/>
+                    </Paper>
+                </Grid>
+                <Grid item xs={6}>
```
Comment (0xVillager) [inline src/components/free-willy/FreeWilly.tsx]: Setting xs to 12 and sm to 6 will make this component responsive.   half screen on small and up, full screen on mobile.  can tweak up to sm/md instead if it looks bad on tablet as well.

**Review by 0xVillager — COMMENTED** [review]
Updated review.

File: src/components/free-willy/FreeWilly.tsx (hunk 5 above)
Comment (0xVillager) [reply src/components/free-willy/FreeWilly.tsx]: e.g.
```typescript 
<Grid container>
    <Grid item xs={12} sm={6}>
       //...
    </Grid>
    <Grid item xs={12} sm={6}>
        //...
     </Grid>
</Grid>
```

File: src/components/free-willy/LiquidationWithdrawals.tsx (hunk 3 above)
Comment (vladjdk) [reply src/components/free-willy/LiquidationWithdrawals.tsx]: No need anymore, the Bid usage was removed with the latest update

**Pull request merged**